protocol/scripts/deploys/helpers.ts

```diff
@@ -63,15 +68,15 @@ export const DEPLOYED_CONTRACTS: {[key: string]: DeployedContracts} = {
   mainnet: {
     // From network/environment
     FRAX: '0x853d955acef822db058eb8505911ed77f175b99e',
-    MULTISIG: "0x4D6175d58C5AceEf30F546C0d5A557efFa53A950",
+    MULTISIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
 
     TEMPLE: '0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7',
     TEMPLE_TEAM_FIXED_PAYMENTS: '0xF7b10A0C780a3906D9A9F3d706EcD2624B6ED84e',
     TEMPLE_TEAM_EPOCH_2: '0xe0Aafcf26576a53Cbec99481607FB53384909C36',
-    TEMPLE_TEAM_EPOCH_3: '', // XXX: FIll in
+    TEMPLE_TEAM_EPOCH_3: '0xf86c2dbd16f05e86bff72ce89b3c2915812e92d0',
```

Comment:
only actual change in this file is adding these 2 addresses, everything else is linting